src/frontend/src/index.less

```diff
@@ -37,4 +37,85 @@ a {
   a[href]:after {
       content: none !important;
   }
+}
+
+.container {
```

Comment:
dette var mye CSS for så låte, man kan ikke klare seg med mye av felles-cssen fra nav-fronend?

Reply:
Godt poeng, ser ikke ut som det trengs fordi det allerede importeres. Fjerner det!